racket/src/cs/c/configure.ac

```diff
@@ -192,6 +192,32 @@ if test "$ARFLAGS" = '' ; then
   ARFLAGS=rc
 fi
 
+if test "${enable_ios}" != "" ; then
```

Comment:
Could more of this be in a file in "racket/src/ac" to be included by both BC and CS configure scripts?